index.js

```diff
@@ -1,35 +1,41 @@
 import { createServer } from "http";
-import { Octokit } from "@octokit/rest";
 import nunjucks from "nunjucks";
+import { OctokitApp } from "./octokitApp.js";
 
 nunjucks.configure({
   autoescape: true,
   watch: true,
 });
 
-const ACCESS_TOKEN = process.env.GITHUB_PERSONAL_ACCESS_TOKEN;
+const httpServer = createServer(async (request, response) => {
+  if (await OctokitApp.middleware(request, response)) return;
```

Comment:
I think it would be good to abstract this out of the callback passed to `createServer` into a named function, but that can come in a later PR.